### PR TITLE
base updates

### PR DESCRIPTION
**Pull request by 0xMakka**
## Description

This PR includes changes discussed in discord to base, i.e.

- Add a "Retire on base" button in the main KD dApp.
- Emphasise the "proof-of-concept" text

### CODE REVIEW

**Review by jabby09 — APPROVED** [review]

**Comment by 0xtapi** [comment]
Just something to add: The "emphasized" part doesn't seem to work for me:
Screenshot 2024-07-30 123039

Still looks like non-emphasized text to me 🤔

**Comment by 0xMakka** [comment]
> Just something to add: The "emphasized" part doesn't seem to work for me: Screenshot 2024-07-30 123039
> 
> Still looks like non-emphasized text to me 🤔

@0xtapi are you using the base preview link above? - https://base-git-makka-base-updates-klimadao.vercel.app/

**Comment by 0xtapi** [comment]
> @0xtapi are you using the base preview link above?

Ah... No, I used the other one and expected to be taken to the same destination. Alright, false alarm 🫡

File: app/components/NavMenu/index.tsx [hunk 1]
```diff
@@ -209,13 +209,22 @@ export const NavMenu: FC<Props> = (props) => {
...
+          <ButtonPrimary
+            target="_blank"
+            href={SQUID_ROUTER_URL}
+            className={styles.baseButton}
+            icon={<BaseLogo className={styles.baseIcon} />}
+            label={<Trans>Bridge to Base</Trans>}
+          />
+          <ButtonPrimary
+            target="_blank"
+            href="https://base-poc.vercel.app"
```
Comment (Atmosfearful) [inline app/components/NavMenu/index.tsx:222]: url is configured:
```suggestion
            href="https://base.klimadao.finance"
```

**Review by Atmosfearful — CHANGES_REQUESTED** [review]

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**